docker-compose.yml

```diff
@@ -24,9 +26,15 @@ services:
       context: client
       dockerfile: Dockerfile
     image: jcayabyab/yw-nlp-frontend
+    environment:
+      - NGINX_PORT=${NGINX_PORT}
+      - BACKEND_PORT=${BACKEND_PORT}
+      - DOLLAR=${DOLLAR}
     ports:
-      - "80:80"
-    restart: always
+      - "${NGINX_PORT}:${NGINX_PORT}"
+    expose:
+      - "${NGINX_PORT}"
+    command: /bin/bash -c "envsubst < /etc/nginx/templates/form.template > /etc/nginx/conf.d/default.conf && cat /etc/nginx/conf.d/default.conf && find /etc/nginx -print && nginx -g 'daemon off;'"
```

Comment:
You can get rid of the `cat` and `find` calls in this pipe, they were only for debugging.

Reply:
done